clients/interface.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2021 The Kubernetes authors.
```

Comment:
Kubernetes author --> OpenYurt author?

Reply:
Thanks for reminding, the license header is changed.